PIL/Image.py

```diff
@@ -583,21 +583,27 @@ def _copy(self):
 
     def _dump(self, file=None, format=None, **options):
```

Comment:
Maybe the builtin file parameter can also be renamed? The underscore indicates this is an internal function, so no one should be using this. Then line 595 can be removed.